devops/actions/aws-ec2/aws-ec2.js

```diff
@@ -150,7 +165,7 @@ async function stop(param_label) {
     for (const reservation of instances.Reservations) {
       for (const instance of reservation.Instances) {
         try {
-          await ec2.terminateInstances({ InstanceIds: [ instance.InstanceId ] }).promise();
+          await ec2.terminateInstances({ InstanceIds: [ instance.InstanceId ] });
```

Comment:
Please update to v3 as well: https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/client/ec2/command/TerminateInstancesCommand/